libraries/api-documenter/package.json

```diff
@@ -15,12 +15,17 @@
   },
   "devDependencies": {
     "@microsoft/node-library-build": "~3.3.0",
+    "@microsoft/node-core-library": "~0.1.3",
+    "@types/chai": "3.4.34",
     "@types/colors": "~1.1.3",
     "@types/es6-collections": "0.5.29",
     "@types/fs-extra": "0.0.37",
     "@types/js-yaml": "~3.9.1",
```

Comment:
Why is this one using ~?

Reply:
Good catch

---
In reply to: [134927225](https://github.com/Microsoft/web-build-tools/pull/330#discussion_r134927225) [](ancestors = 134927225)